### PR TITLE
Add cdc race codes

### PR DESCRIPTION
**Pull request by losborne**
Bonnie hardcodes 'CDC Race' for Race/Ethnicity rather than using the correct 'cdcrec'.  This is a workaround to have everything calculating/matching between Bonnie/Backend calculation until we can properly convert Bonnie to use 'cdcrec', which is a more substantial change.

Pull requests into cqm-converter require the following. Submitter and reviewer should :white_check_mark: when done. For items that are not-applicable, note it's not-applicable ("N/A") and :white_check_mark:.

**Submitter:**
- [x] This pull request describes why these changes were made.
- [x] Internal ticket for this PR: https://jira.mitre.org/browse/BONNIE-1627
- [x] Internal ticket links to this PR
- [x] Code diff has been done and been reviewed
- [x] Tests are included and test edge cases
- [x] Tests have been run locally and pass
- [x] If applicable, the library version number in `cqm-converter.gemspec` has been updated N/A

**Bonnie Reviewer:**

Name: @hossenlopp 
- [x] Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose
- [x] The tests appropriately test the new code, including edge cases
- [x] You have tried to break the code

**Cypress Reviewer:**

Name: @dczulada 
- [x] Change will not affect Cypress

### CODE REVIEW

File: lib/cqm/converter/hds_record.rb [hunk 1]
```diff
@@ -72,7 +72,11 @@ def to_qdm(record)
       ethnicity = record.ethnicity
       if ethnicity
         # See: https://phinvads.cdc.gov/vads/ViewCodeSystem.action?id=2.16.840.1.113883.6.238
-        code = QDM::Code.new(ethnicity['code'], 'cdcrec', ethnicity['name'], '2.16.840.1.113883.6.238')
+        # Bonnie currently uses 'CDC Race' instead of the correct 'cdcrec'.  This duplicate code is here as a temporary
+        # workaround until the larger change of making bonnie use 'cdcrec' can be implemented.
+        # Same change is present in `race` below.
+        code = QDM::Code.new(ethnicity['code'], 'CDC Race', ethnicity['name'], '2.16.840.1.113883.6.238')
+        # code = QDM::Code.new(ethnicity['code'], 'cdcrec', ethnicity['name'], '2.16.840.1.113883.6.238')
```
Comment (hossenlopp) [inline lib/cqm/converter/hds_record.rb]: It doesn't look like this is adding the duplicate code. It looks like the single wrong code is being added.

Comment (losborne) [reply lib/cqm/converter/hds_record.rb]: Ah good catch, stale comment.  Will update
